p2p/Cargo.toml

```diff
@@ -29,6 +29,7 @@ subtle-encoding = { version = "0.5" }
 thiserror = "1"
 x25519-dalek = "1.1"
 zeroize = "1"
+pipe = { version = "0.4.0", features = ["bidirectional"] }
```

Comment:
This dependency should eventually be listed under the `[dev-dependencies]` section if it's only used in tests.